constants.py

```
@@ -100,45 +100,51 @@ class OfficeType(Enum):
-    "\n[System Notice] 已收到上传的{type_desc}：{original_name}（后缀：{file_suffix}），"
-    " 文件已保存到工作区。"
-    " 如果用户请求依赖这个上传文件，先调用 `read_file`，再调用 `create_document`"
-    " 或 `create_office_file`。"
-    " 在至少读取一次上传源文件之前，不要先创建新文档。"
-    " 如果 `read_file` 返回文件不存在或路径非法，不要调用网络搜索；"
-    " 直接请用户重新上传文件或提供正确的本地路径。"
-    " 仅在任务仍不清楚时，才先追问用户。"
-    " 如果要先给用户一句过渡说明，也请使用中文，不要先输出英文说明。"
-    " 对于复杂 Word 输出，优先使用有状态文档工具，而不是 `create_office_file`。"
+    "\n[System Notice] [ACTION REQUIRED] 已收到上传文件\n"
+    "- 文件类型：{type_desc}\n"
```

Ruff (0.15.6)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)